provider/kv.go

```diff
@@ -162,6 +162,9 @@ func (provider *Kv) list(keys ...string) []string {
 func (provider *Kv) listServers(backend string) []string {
 	serverNames := provider.list(backend, "/servers/")
 	return fun.Filter(func(serverName string) bool {
+		if _, err := provider.kvclient.Get(fmt.Sprint(serverName, "/url")); err != nil {
```

Comment:
Re: error message: I'd use something like:

`log.Errorf("Failed to retrieve value for key %s: %s", key, err)`

`key` can be an extracted variable because we might need it up to two times (once for the `Get` and then again for the error message).